<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PreCC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="pathLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shots Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="shotsPathBox"/>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="subStatusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>Resize each render image</string>
        </property>
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sizeBox">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Resize each render image</string>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="createMovButton">
        <property name="toolTip">
         <string>Create a .mov file by rendering all the frames</string>
        </property>
        <property name="text">
         <string>Create .mov</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Render selected shots</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createMovButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>sizeBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createMovButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
